pkg/transferer/load.go

```diff
@@ -127,7 +129,22 @@ func (c *LoadChecker) checkMetricsLoad() (LoadStatus, error) {
 	log.Debug().Msg("Started check load status")
 	loadStatus := LoadStatusOK
 	for _, t := range c.thresholds {
-		value, err := c.getMetricCurrentValue(t)
+		value, err := 0.0, error(nil)
+
+		switch t.Key {
+		case ThresholdMYRAM:
+			rms := runtime.MemStats{}
+			runtime.ReadMemStats(&rms)
+			mv, me := mem.VirtualMemory()
```

Comment:
We don't need to import package just for one function. Please write this functionality by yourself